components/src/components/convert/convert.tsx

```diff
@@ -493,10 +510,14 @@ export class Convert {
                         </p>
                         <p>
                             The code you submitted was valid, but { this.conversionTool.name } was unable to
-                            convert it completely, so a partial conversion has been provided for you. See below for details.
+                            convert it completely, so a partial conversion has been provided for you. See below
+                            for details. Please also check the <a href={this.conversionTool.issueURL}>known issues</a>
+                            or report a <a href={this.conversionTool.issueURL}>new issue</a> if you believe this 
```

Comment:
This is annoying, but because of the way whitespace is stripped around tags, there needs to be an explicit space around the `a` tag here to prevent it from being rendered as "known issuesor". 

![image](https://user-images.githubusercontent.com/274700/100802801-e7399900-33de-11eb-829f-3d51a1fe6225.png)

This should do it, though:

```suggestion
                            for details. Please also check the <a href={this.conversionTool.issueURL}>known issues</a> or
                            report a <a href={this.conversionTool.issueURL}>new issue</a> if you believe this 
```